postgres-operator.go

```diff
@@ -181,7 +181,7 @@ func main() {
 
 	cluster.InitializeAutoFailover(Clientset, crdClient, namespaceList)
 
-	operatorupgrade.OperatorUpgrade(Clientset, crdClient, namespaceList)
+	operatorupgrade.OperatorUpdateCRPgoVersion(Clientset, crdClient, namespaceList)
```

Comment:
Is there a reason for the function name change? Not sure I follow what This is doing.

Reply:
it expands to "Operator Update Custom Resource PGO Version." I changed it to make it more distinct than prior name (which generally referred to several different upgrades or update) - the function is commented at its definition.